dDatabase/GameDatabase/GameDatabase.h

```diff
@@ -26,13 +26,8 @@
 #include "IBehaviors.h"
 #include "IUgcModularBuild.h"
 
-namespace sql {
-	class Statement;
-	class PreparedStatement;
-};
-
 #ifdef _DEBUG
-#  define DLU_SQL_TRY_CATCH_RETHROW(x) do { try { x; } catch (sql::SQLException& ex) { LOG("SQL Error: %s", ex.what()); throw; } } while(0)
+#  define DLU_SQL_TRY_CATCH_RETHROW(x) do { try { x; } catch (std::exception& ex) { LOG("SQL Error: %s", ex.what()); throw; } } while(0)
```

Comment:
Nit: If this is no longer exclusively catching SQL errors, the logging statement should probably specify it's a generic exception.

Reply:
SQL is SQL it's just what flavor of SQL that it is
Mariadb(MySQL) or SQLite

Reply:
Ah, makes sense.